### ISSUE TITLE
Hidden "people" show up as commas...

### ISSUE THREAD

**Issue by Snille** [pelaxa/MMM-ImmichSlideShow#18]
Added some old "class photos" and noticed this...
![image](https://github.com/pelaxa/MMM-ImmichSlideShow/assets/1011699/33e0b347-a3e9-41dd-9f59-58be23fba907)
The people in the picture that is not named, is showing up as commas... :)

**Comment by pelaxa** [comment]
Yes, I noticed that as well. Immich tries to name people from left to right, so without some indicator, it is hard to know which person the name belongs to.

also trying to avoid making that info section too big so it takes over more of the screen space. It has to be a fine balance. 

Open to suggestions.
